tests/TestCase/Database/QueryTests/CaseExpressionQueryTest.php

```diff
@@ -332,15 +379,16 @@ public function testBindValues(string $when, int $result): void
                         ->setReturnType('integer'),
```

Comment:
Same here

Reply:
This call can be dropped, as it's only used with ORM queries.

Reply:
Could pull this and the input casting tests into a test case that is for the casting features of ORM\Query :thinking: